cvat/apps/dataset_manager/tests/test_rest_api_formats.py

```diff
@@ -345,6 +352,13 @@ def _delete_project(self, project_id, user):
         self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
         return response
 
+    @staticmethod
+    def _save_file_from_response(response, file_zip_name):
+        if response.status_code == status.HTTP_200_OK:
+            content = BytesIO(b"".join(response.streaming_content))
```

Comment:
The `BytesIO` object seems unnecessary here.

Reply:
removed it